### DISCUSSION TITLE
It's about power apps - my custom form won't open because of github?

### DISCUSSION THREAD

**Discussion by joaoandrebs** [community/community#63007] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Hello!

I created a repository to enter the data of an "app" that I created to remember the codes --- after a while I was warned that github is not allowed by my company and it deletes all the repositories that I had linked Power Apps to - --

All the custom forms (apps) that I had created in the github repositories were able to open normally for editing on the canvas, but a specific one is not opening because it asks me to enter my repository - but this one has already been deleted -

Does anyone know how to solve this problem? I can no longer edit.

Note: My custom form comes from an SP list.

[
![image](https://github.com/community/community/assets/127403085/8c381679-c2bb-4a01-bbb4-208c298bac41)
](url)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by SAMIULLAH-KALHORO-DEV** [comment]
It seems like you're encountering issues with accessing a specific custom form (app) in Power Apps because it was linked to a GitHub repository that has been deleted due to your company's policies. This situation can indeed cause challenges in accessing and editing the form. Here's what you can try to resolve the issue:

1. **Disconnect GitHub Repository:** Since your GitHub repository was deleted, the connection between the custom form and the repository has been broken. You will need to disconnect the deleted repository from the form to clear the error message.

2. **Check Form Connections:** Go to the Power Apps interface and open the custom form that's giving you the error. Look for any references or connections to the deleted GitHub repository. You need to remove these references.

3. **Reconnect or Rebuild:** If your custom form was tightly integrated with the GitHub repository and cannot be opened without it, you might need to consider recreating or rebuilding the form. You can connect it to a new repository or remove the GitHub dependency altogether.

4. **Recreate from SP List:** Since you mentioned that your custom form comes from an SP (SharePoint) list, you could also try recreating the form using the data from the SP list directly. Power Apps provides tools to create forms that are directly connected to SharePoint lists without needing a GitHub repository.

5. **Check with IT or Admins:** If your company's policies prohibit the use of GitHub and have resulted in the deletion of repositories, it's advisable to check with your company's IT or administrators for guidance. They might have insights into alternative solutions or provide assistance in recreating the form.

6. **Power Apps Support:** If you're unable to resolve the issue using the steps above, consider reaching out to Power Apps support for assistance. They can provide specific guidance and recommendations based on your situation.

Remember that each situation can be unique, and the steps you need to take might vary based on the specifics of your custom form and company policies. It's important to clear any dependencies on the deleted repository and find alternative solutions to ensure continued access and editing of your custom form.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @joaoandrebs ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
